config/HH_bbWW/Run3_2022/samples.yaml

```diff
@@ -618,4 +618,21 @@ VBFHHto2B2Vto2L2Nu_CV-m2p12_C2V-3p87_C3-m5p96:
   sampleType: HHnonRes
   generator: madgraph
   #crossSection: To_be_added
-############## Finished non -Resonant Signal samples)##############
\ No newline at end of file
+############## Finished non -Resonant Signal samples)##############
+############## Data ##############
+EGamma_Run2022C:
+  sampleType: data
+EGamma_Run2022D:
+  sampleType: data 
+Muon_Run2022C:
+  sampleType: data  
+Muon_Run2022D:
+  sampleType: data  
+SingleMuon_Run2022C:
+  sampleType: data  
+DoubleMuon_Run2022C:
+  sampleType: data  
+MuonEG_Run2022C:
+  sampleType: data  
+MuonEG_Run2022D:
+  sampleType: data  
```

Comment:
- data datasets should be defined in `config/Run3_2022/samples.yaml`
- define all datasets that we can potentially use in bbWW or bbtautau analyses